tasks/request_certificate.yml

```diff
@@ -5,6 +5,34 @@
     path: "{{ certbot_certificate_path }}"
   register: cert_status
 
-- name: Initiate certificate request
-  command: "{{ certbot_request_command }}"
+- block:
+    - name: Determine if port 80 is open to the world
```

Comment:
`firewall-cmd --query-rich-rule='rule family="ipv4" source address="0.0.0.0./0" service name="http" accept'`
will likely be able to dispense with all of the shell, and return a proper status code reflecting the state of the rich rule

Reply:
This is nice - thank you! I'll make the update.